components/grid/grid.css

```diff
@@ -0,0 +1,55 @@
+.grid-two-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
Instead, you can use only two columns as the name suggest. 

There won't be any need to add grid areas

Reply:
Sure, I'll look into it